src/main/java/task/Event.java

```diff
@@ -0,0 +1,24 @@
+package task;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+
+public class Event extends Task{
+    protected LocalDate date;
+
+    public Event(String command, String date) {
+        super(command);
+        this.date = LocalDate.parse(date);
+    }
+
+    @Override
```

Comment:
For consistency's sake, perhaps you could consider writing a comment or Javadoc here to explain the output (the format, how to interpret it etc), just like what you did for Deadline.